app/bundles/CampaignBundle/Form/Type/CampaignListType.php

```diff
@@ -60,38 +62,39 @@ public function configureOptions(OptionsResolver $resolver)
                     $choices   = [];
                     $campaigns = $this->model->getRepository()->getPublishedCampaigns(null, null, true, $this->canViewOther);
                     foreach ($campaigns as $campaign) {
-                        $choices[$campaign['id']] = $campaign['name'];
+                        $choices[$campaign['name']] = $campaign['id'];
                     }
 
                     //sort by language
-                    asort($choices);
+                    ksort($choices);
 
                     if ($options['include_this']) {
                         $choices = ['this' => $options['this_translation']] + $choices;
                     }
 
                     return $choices;
                 },
-                'empty_value'      => false,
-                'expanded'         => false,
-                'multiple'         => true,
-                'required'         => false,
-                'include_this'     => false,
-                'this_translation' => 'mautic.campaign.form.thiscampaign',
+                'choices_as_values' => true,
+                'empty_value'       => false,
+                'expanded'          => false,
+                'multiple'          => true,
+                'required'          => false,
+                'include_this'      => false,
+                'this_translation'  => 'mautic.campaign.form.thiscampaign',
             ]
         );
     }
 
     /**
      * @return string
      */
-    public function getName()
+    public function getBlockPrefix()
     {
-        return 'campaign_list';
+        return self::class;
```

Comment:
I don't see this form name to be used in JS nor CSS, but I can see this form be used in 8 other forms. Like `CampaignEventAddRemoveLeadType`. Remember the alias must be replaced in all other places where it is used with FQCN. Check all other form aliases that they aren't used elsewhere.

The `getBlockPrefix()` method can be removed.